specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -3406,18 +3406,1180 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/invoiceSections/{invoiceSectionName}/initiateTransfer": {
+      "post": {
```

Comment:
Why POST to initiate a transfer instead of a PUT to create a "/transfers" resource?  POST means a user will not be able to perform this via a template deployment and doesn't give a clear link between the POST call and the "transfers" resource that is created for it

Reply:
I have the same question. Can you reply to this?

Reply:
The use case for our initiateTransfer API resembles POST action than PUT. As per current implementation (for preview versions), an email is sent to recipient each time a transfer request is created by initiator, We are planning to have more discussion on it in Scheduled meeting.